_includes/projects-feature.html

```diff
@@ -1,30 +0,0 @@
-<div class="project-container">
```

Comment:
Removed this entire file as I found a way to use the project.html include in both places